src/components/TextField/TextField.tsx

```diff
@@ -24,10 +24,8 @@ export const TextField: React.FC<Props> = ({
   required = false,
   onChange = () => {},
 }) => {
-  // generage a unique id once on component load
   const [id] = useState(() => `${name}-${getRandomDigits()}`);
 
-  // To show errors only if the field was touched (onBlur)
   const [touched, setTouched] = useState(false);
   const hasError = touched && required && !value;
```

Comment:
```suggestion
  const hasError = touched && required && !value.trim();
```
this comment not fixed
https://github.com/mate-academy/react_movies-list-add-form/pull/1707#pullrequestreview-1553645900